api/graphite_req.go

```diff
@@ -108,7 +108,7 @@ func (rp ReqsPlan) PointsReturn(planMDP uint32) uint32 {
 }
 
 func (rp ReqsPlan) List() []models.Req {
-	l := make([]models.Req, rp.cnt, 0)
+	l := make([]models.Req, 0, rp.cnt)
```

Comment:
ah, is this why we had requests with Archinterval 0 ?

Reply:
Yes, we just had the `len` and `cap` reversed